src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -342,7 +349,7 @@ const BudgetCategoriesTableContainer = (props) => {
             fetchBudgetSpendingCallback();
         }
         changeCurrentPage(1);
-    }, [pageSize, sort, order, defCodes, overview, allAwardTypeTotals]);
+    }, [pageSize, sort, order, overview, allAwardTypeTotals]);
```

Comment:
We could add `publicLaw` as a dependency to ensure the table updates when the filter changes, although `allAwardTypeTotals` may handle that indirectly